chia/clvm/spend_sim.py

```diff
@@ -369,20 +366,16 @@ async def get_additions_and_removals(self, header_hash: bytes32) -> Tuple[List[C
             filter(lambda br: br.header_hash == header_hash, self.service.block_records)
         )[0]
         block_height: uint32 = selected_block.height
-        additions: List[CoinRecord] = await self.service.mempool_manager.coin_store.get_coins_added_at_height(
-            block_height
-        )  # noqa
-        removals: List[CoinRecord] = await self.service.mempool_manager.coin_store.get_coins_removed_at_height(
-            block_height
-        )  # noqa
+        additions: List[CoinRecord] = await self.service.coin_store.get_coins_added_at_height(block_height)  # noqa
```

Comment:
you could just use `self.coin_store` here too, right?
and below. Maybe that would let you remove the `# noqa`